src/components/TopRow.tsx

```diff
@@ -1,38 +1,45 @@
+import Link from "next/link";
 import { Box, HStack, Text, Icon } from "@chakra-ui/react";
 import { CustomConnectButton } from "components/CustomConnectButton";
 import { ColorModeSwitcher } from "components/ColorModeSwitcher";
-import { ImStatsBars, ImTable } from "react-icons/im";
-import { CgCardSpades } from "react-icons/cg";
+import { ImStatsBars as StatsIcon, ImTable as TableIcon } from "react-icons/im";
+import { CgCardSpades as CardIcon } from "react-icons/cg";
 
 const iconSize = "2rem";
+const iconMarginR = "1rem";
 
 export const TopRow = () => {
   return (
     <>
-      <HStack justify="flex-end">
+      <HStack justify="flex-end" margin="1rem">
```

Comment:
with chakra (and tailwind) you can do `m="1rem"`, i know it violates my no abbreviation rule, but i do like that personally. (you can use mt, mb, my, ml, etc.) and do you prefer the `rem` dimension? I'm not sure what my preference. I think i'm a fan of  m={4} (that will give a us 4 chakra "spacing" and if for some reason you wanted to adjust the whole app for a breakpoint you could do it in the theme.